### DISCUSSION TITLE
[Feature] Mark an asset for inclusion in an album once uploaded

### DISCUSSION THREAD

**Discussion by laxmanpradhan** [immich-app/immich#11105] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

For items that are local (such as on the phone) but not yet uploaded to the server (large videos for example that will need WiFi at a later time), it would be nice to be able to add it to an album on the mobile app and then when the asset is finally uploaded to the server, the server knows to include it in to the album.

Currently, it will simply fail and say local assets can't be added to an album.

This is useful in cases where you have many photos and videos that were taken and you want to start organizing them or sharing them as soon as they are taken.

Small but nice QoL feature.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678
